mlflow/gateway/client.py

```diff
@@ -137,7 +137,9 @@ def search_routes(self, page_token: Optional[str] = None) -> PagedList[Route]:
         return PagedList(routes, next_page_token)
 
     @experimental
-    def create_route(self, name: str, route_type: str, model: Dict[str, Any]) -> Route:
+    def create_route(
+        self, name: str, route_type: str = None, model: Dict[str, Any] = None
+    ) -> Route:
```

Comment:
/ai add `Optional` to type hints on `route_type` and `model`.

Reply:
the bot types faster than me.

Reply:
No chance to complete with him for speed :)

Reply:
Added `Optionals`. To confirm, this won't be a breaking change then like a Scala.Option right? The usage would still be `route_type="llm/v1/completions` or `route_type=None`

Reply:
No, it won't break anything :)